ci/container-images/permaplant-mdbook/Dockerfile

```diff
@@ -2,5 +2,5 @@ FROM rust:1.67.1-slim
 
 RUN apt update && apt install -y libpq-dev curl pkg-config ca-certificates build-essential make perl gcc libc6-dev
 RUN cargo install mdbook mdbook-mermaid
-RUN cargo install --git https://github.com/ElektraInitiative/mdbook-generate-summary mdbook-generate-summary
+RUN cargo install --git https://github.com/ElektraInitiative/mdbook-generate-summary mdbook-generate-summary --locked
 RUN cargo install mdbook-linkcheck
```

Comment:
You can move this line to the above install as well to reduce the number of layer in the Dockerfile.
So you could do `RUN cargo install mdbook mdbook-mermaid mdbook-linkcheck`.
But still its good.

Reply:
I was thinking about that, but didnt touch it since it was always like that, will do it next time I change something in this file.